src/USER-MISC/fix_bond_react.cpp

```diff
@@ -2322,6 +2324,8 @@ void FixBondReact::update_everything()
                   nspecial[atom->map(update_mega_glove[jj+1][i])][1]++;
                   nspecial[atom->map(update_mega_glove[jj+1][i])][2]++;
                 }
+                if (nspecial[atom->map(update_mega_glove[jj+1][i])][2] > atom->maxspecial)
+                  error->one(FLERR,"Bond/react special bond generation overflow");
```

Comment:
Need to add these errors messages with an explanation to the `fix_bond_react.h` file.

Reply:
...and to `doc/src/Errors_messages.txt`